runtime/src/snapshot_utils.rs

```diff
@@ -1293,7 +1291,7 @@ fn spawn_unpack_snapshot_thread(
     Builder::new()
         .name(format!("solUnpkSnpsht{thread_index:02}"))
         .spawn(move || {
-            streaming_unpack_snapshot(
+            hardened_unpack::streaming_unpack_snapshot(
```

Comment:
doesn't matter much, but this one is used in normal (non-dcou) stuff right? Just unclear why this specific use was removed.